server/src/main/java/org/apache/cassandra/sidecar/cluster/CassandraAdapterDelegate.java

```diff
@@ -645,12 +645,8 @@ public void handleNotification(Notification notification, Object handback)
                 switch (type)
                 {
                     case JMXConnectionNotification.OPENED:
-                        // Do not notify here as we may not have set up our own delegate yet
-                        // Instead, run the JMX Health Check, which will notify once we have
-                        // created or updated the adapter.
-                        jmxHealthCheck();
+                        LOGGER.info("JMX connection opened");
```

Comment:
Can we move this to debug? We already log connections elsewhere and this would just be redundant.
```suggestion
                        LOGGER.debug("JMX connection opened");
```

Reply:
updated